packages/driver/src/cypress/cy.ts

```diff
@@ -581,6 +565,22 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
 
         // If the runner can communicate, we should setup all events, otherwise just setup the window and fire the load event.
         if (isRunnerAbleToCommunicateWithAUT) {
+          if (this.Cypress.isBrowser('webkit')) {
```

Comment:
Moved this logic to only run if the runner is able to communicate with the AUT.

We may want to add this patch to the injection code for cy.origin when we enable it.